exec/java-exec/src/main/java/org/apache/drill/exec/util/record/RecordBatchStats.java

```diff
@@ -279,4 +324,19 @@ private static void logBatchStatsMsg(RecordBatchStatsContext batchStatsContext,
     }
   }
 
+  private static String toString(BatchIOType ioType) {
+    Preconditions.checkNotNull(ioType, "The record batch IO type cannot be null");
+
+    switch (ioType) {
```

Comment:
'case' is generally indented one level into the 'switch'. 

Also, Java allows Enums to have constructors and methods that  can be used to associate a string with an enum value. This makes modification of enum names and attributes easier. See https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html